app/src/main/java/com/sun_asterisk/moviedb_50/data/source/remote/fetchjson/ParseJsonToGenres.kt

```diff
@@ -0,0 +1,34 @@
+package com.sun_asterisk.moviedb_50.data.source.remote.fetchjson
+
+import com.sun_asterisk.moviedb_50.data.model.Genres
+import org.json.JSONException
+import org.json.JSONObject
+
+class ParseJsonToGenres {
+    fun parseJsonToData(jsonObject: JSONObject): MutableList<Genres>? {
+        val genresList: MutableList<Genres> = mutableListOf()
```

Comment:
biến địa phương không khai báo kiểu 
```suggestion
        val genresList = mutableListOf()
```